app/javascript/__tests__/components/__snapshots__/OpenHouses.test.tsx.snap

```diff
@@ -0,0 +1,40 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`OpenHouses displays OpenHouses with Listing containing Open_Houses 1`] = `
+<section
+  className="aside-block"
+>
+  <h4
+    className="text-caps-underline undefined"
+  >
+    Open Houses
+  </h4>
+  <div
+    className="false"
+  >
+    <p
+      className="text text-gray-800 pb-2 flex justify-between items-center"
+    >
+      <span
+        className="inline-block text-tiny uppercase"
```

Comment:
any ideas on how best to not make the dates all uppercase?

Reply:
yup! There's a PR in bloom for customization of the component with that uppercase field. Whatever strategy gets merged will support title-casing that date.

Reply:
awesome, thanks!